test/integration/admin/cms/switches_controller_test.rb

```diff
@@ -31,14 +31,4 @@ def setup
     assert_response :success
     assert provider.settings.switches[switch_name].visible?
   end
-
-  test "show a  switch" do
```

Comment:
This test was added as an extraction of the Rails 5 branch in https://github.com/3scale/porta/pull/1576, and now it is removed again... :confused: I am very confused with this.

Reply:
That is why your review is valuable :)